webapp/channels/src/actions/user_actions.ts

```diff
@@ -340,12 +337,14 @@ export async function loadProfilesForGM() {
                 value: 'true',
             });
         }
-
-        const getProfilesAction = UserActions.getProfilesInChannel(channel.id, 0, Constants.MAX_USERS_IN_GM);
-        queue.add(() => dispatch(getProfilesAction));
+        if (userIds.size === 0) {
+            channelUsersToLoad.push(channel.id);
+        }
     }
 
-    await queue.onEmpty();
+    if (channelUsersToLoad.length > 0) {
+        await dispatch(UserActions.getProfilesInGroupChannels(channelUsersToLoad));
```

Comment:
do we need to wait here?

Reply:
Yeah we need to ensure the users exist in the redux store before we call `loadCustomEmojisForCustomStatusesByUserIds`. That custom emoji's function checks the custom status for all those user IDs